js/newTabPage.js

```diff
@@ -9,6 +11,15 @@ const newTabPage = {
   imagePath: path.join(window.globalArgs['user-data-path'], 'newTabBackground'),
   blobInstance: null,
   reloadBackground: function () {
+    const newTabUrl = settings.get('newTabUrl')
+    if (newTabUrl === 'blank') {
```

Comment:
If the setting has never been modified, settings.get will return undefined, which this also needs to check for.